pisa/stages/flux/mceq_barr.py

```diff
@@ -287,47 +293,51 @@ def setup_function(self):
             true_log_energy = np.log(container["true_energy"])
             true_abs_coszen = np.abs(container["true_coszen"])
             nu_flux_nominal = container["nu_flux_nominal"]
+            nu_flux_mceq = container["nu_flux_mceq"]
             gradients = container["gradients"]
             nubar = container["nubar"]
 
             #
             # Nominal flux
             #
 
-            if not self.use_honda_nominal_flux :
+            # Evaluate splines to get nominal flux
+            # Need to correctly map nu/nubar and flavor to the output arrays
 
-                # Evaluate splines to get nominal flux
-                # Need to correctly map nu/nubar and flavor to the output arrays
+            # Note that nominal flux is stored multiple times (once per Barr parameter)
+            # Choose an arbitrary one to get the nominal fluxes
+            arb_gradient_param_key = self.gradient_param_names[0]
 
-                # Note that nominal flux is stored multiple times (once per Barr parameter)
-                # Choose an arbitrary one to get the nominal fluxes
-                arb_gradient_param_key = self.gradient_param_names[0]
+            # nue(bar)
+            nu_flux_mceq[:, 0] = self.spline_tables_dict[arb_gradient_param_key]["nue" if nubar > 0 else "nuebar"](
+                true_abs_coszen,
+                true_log_energy,
+                grid=False,
+            )
 
-                # nue(bar)
-                nu_flux_nominal[:, 0] = self.spline_tables_dict[arb_gradient_param_key]["nue" if nubar > 0 else "nuebar"](
-                    true_abs_coszen,
-                    true_log_energy,
-                    grid=False,
-                )
+            # numu(bar)
+            nu_flux_mceq[:, 1] = self.spline_tables_dict[arb_gradient_param_key]["numu" if nubar > 0 else "numubar"](
+                true_abs_coszen,
+                true_log_energy,
+                grid=False,
+            )
 
-                # numu(bar)
-                nu_flux_nominal[:, 1] = self.spline_tables_dict[arb_gradient_param_key]["numu" if nubar > 0 else "numubar"](
+            # nutau(bar)
+            # Currently setting to 0 #TODO include nutau flux (e.g. prompt) in splines
```

Comment:
This comment seems to relate to what the `include_nutau_flux` initialisation parameter effectuates. Could this be added to the service's docstring instead (which has unfortunately been missing both this and the `use_honda_nominal_flux` parameter for a while)?